android/src/main/java/com/pdftron/reactnative/views/DocumentView.java

```diff
@@ -3879,6 +3898,21 @@ public void showCropDialog() {
         }
     }
 
+    public void openOutlineList() {
+        if (isBookmarkListVisible) {
+            mPdfViewCtrlTabHostFragment.onOutlineOptionSelected(1);
+        } else {
+            mPdfViewCtrlTabHostFragment.onOutlineOptionSelected(0);
+        }
+    }
+
+    public void openLayersList() {
+        if (isPdfLayersListVisible) {
```

Comment:
- [ ] null check for pdfviewctrl
- [ ] no need to check visible, the only reason the other ones need to check visible is because it uses index